packages/stark-core/src/logging/services/logging.service.spec.ts

```diff
@@ -0,0 +1,304 @@
+"use strict";
+
+import Spy = jasmine.Spy;
+import { Action, Store } from "@ngrx/store";
+import { Observable } from "rxjs/Observable";
+import "rxjs/add/observable/of";
+import "rxjs/add/observable/throw";
+import { Serialize } from "cerialize";
+
+import { StarkLoggingActionTypes, FlushLogMessages } from "../../logging/actions/index";
+import { StarkLoggingServiceImpl } from "./logging.service";
+import { StarkApplicationConfig, StarkApplicationConfigImpl } from "../../configuration/entities/index";
+import { StarkLogging, StarkLoggingImpl, StarkLogMessage, StarkLogMessageImpl, StarkLogMessageType } from "../../logging/entities/index";
+import { StarkBackend } from "../../http/entities/backend";
+import { StarkCoreApplicationState } from "../../common/store/starkCoreApplicationState";
+// import {StarkXSRFService} from "../../xsrf";
+// import {UnitTestingUtils} from "../../test/unit-testing";
+
+describe("Service: StarkLoggingService", () => {
+	let appConfig: StarkApplicationConfig;
+	let mockStore: Store<any>;
+	// let mockXSRFService: StarkXSRFService;
```

Comment:
Why is the mockXSRFService not mocked? We can mock it, and in fact we should so we can complete the unit tests